source/hardware/controllers/hercules_p32_dj.rst

```diff
@@ -97,4 +98,4 @@ Other controls
 -  **Shift + cue**: jump to beginning of track and stop
 -  **Shift + play**: toggle reverse playback
 -  **Shift + load**: ejects a track from the deck
--  **EQ knobs, volume faders, crossfader, headphone button, play, cue, sync, and load** all behave as labeled.
+-  **EQ knobs, volume faders, crossfader, headphone button, play, cue, sync, and load** all behave as labeled. Dry/wet knob can optionally be used as pregain (see dryWetKnobAsPregain variable)
```

Comment:
```suggestion
-  **EQ knobs, volume faders, crossfader, headphone button, play, cue, sync, and load** all behave as labeled. Dry/wet knob can optionally be used as pregain (see `dryWetKnobAsPregain` variable)
```

Not relevant to this PR, but it would be cool if you could update the mapping documentation to use the `:hwlabel:` directive like we do in the DDJ-400/Roland DJ-505 docs, i. e. Mark labels found on the controller like this:

    :hwlabel:`CUE`

Reply:
ping